index.html

```diff
@@ -40,7 +40,176 @@ <h4>I'am a Software Developer</h4>
+          </nav>
+        </div>
+
+        <div class="pj1">
+          <h3 class="tproj">Profesional Art Printing Data</h3>
+          <p class="desproj">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="media2">
+            <li><div class="cont1">html</div></li>
+            <li><div class="cont1">bootstrap</div></li>
+            <li><div class="cont1">Ruby</div></li>
+          </ul>
+          <nav id="contbut">
+            <button class="bigbutton" class="button2" type="button">See Project</button>
+          </nav>
+        </div>
+
+        <div class="pj1">
+          <h3 class="tproj">Profesional Art Printing Data</h3>
+          <p class="desproj">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="media2">
+            <li><div class="cont1">html</div></li>
+            <li><div class="cont1">bootstrap</div></li>
+            <li><div class="cont1">Ruby</div></li>
+          </ul>
+          <nav id="contbut">
+            <button class="bigbutton" class="button2" type="button">See Project</button>
+          </nav>
+        </div>
+
+        <div class="pj1">
+          <h3 class="tproj">Profesional Art Printing Data</h3>
+          <p class="desproj">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="media2">
+            <li><div class="cont1">html</div></li>
+            <li><div class="cont1">bootstrap</div></li>
+            <li><div class="cont1">Ruby</div></li>
+          </ul>
+          <nav id="contbut">
+            <button class="bigbutton" class="button2" type="button">See Project</button>
+          </nav>
+        </div>
+
+        <div class="pj1">
+          <h3 class="tproj">Profesional Art Printing Data</h3>
+          <p class="desproj">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="media2">
+            <li><div class="cont1">html</div></li>
+            <li><div class="cont1">bootstrap</div></li>
+            <li><div class="cont1">Ruby</div></li>
+          </ul>
+          <nav id="contbut">
+            <button class="bigbutton" class="button2" type="button">See Project</button>
+          </nav>
+        </div>
+
+      </div>
+    </section>
+
+    <section class="about">
+      <section name="first_about">
+        <h1 id="t_about">About me</h1>
+        <p id="c_about">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <div id="fit_cont">
+         <nav>
+            <button id="button_about" type="button">Get My Resume</button>
+          </nav>
```

Comment:
- [ ] Again, please do not use `<nav>` tag to wrap your buttons. Instead, you can use the usual `div` tag in order to add more styles maybe 😃 👍